frontend/src/components/PhaseSelectionBox/PhaseSelectionBox.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+import { Phase } from "../../models/phase";
+import { PHASES } from "../../Phases";
+import {
+  getCurrentPhase,
+  setCurrentPhase,
+  setCompletedPhase,
+} from "../../service/phaseService";
+
+import "./PhaseSelectionBox.css";
+
+function PhaseSelectionBox(this: any) {
+  // start on sandbox mode
+  const [currPhase, setCurrPhase] = useState<Phase>(PHASES[3]);
```

Comment:
Change this to just be a number representing the index to the current phase. eg. `0` for phase 0, `3` for sandbox.